src/components/PollList/pollList.js

```diff
@@ -12,8 +12,11 @@ import SuccessMessage from "../../utils/successMessage/successMessage";
 import { optionVoteCount } from "../../utils/voteCountUtils";
 import deletePoll, { emptyDeletePollSuccessStatus } from "../../redux/delete/actions/deletePoll";
 
-
 const PollList = () => {
+  const pollQuestion = useSelector((state) => state.pollList);
+  const userDetails = useSelector((state) => state.login.userLogin);
```

Comment:
why reducer name is userLogin and its ref name is userDetails? @PuniaMohit choose name wisely